code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -1056,18 +1056,22 @@
 	max_range = CONFIG_GET(number/combat_define/long_shell_range)
 
 /datum/ammo/rocket/wp/quad/on_hit_mob(mob/M,obj/item/projectile/P)
+	message_admins("ON _ HIT _ MOB")
```

Comment:
leaving stuff in like this makes it do not test merge

Reply:
👍

Reply:
The admins are very angery.